README.md

```diff
@@ -2,19 +2,20 @@
 
 The Kani Rust Verifier is a bit-precise model checker for Rust.
 
+Kani is particularly useful for verifying unsafe code in Rust, where many of the language's usual guarantees are no longer checked by the compiler.
 Kani verifies:
- * Memory Safety -- even in unsafe Rust code
- * User-specified assertions
- * Absence of panics
- * Absence of some classes of undefined behavior
+ * Memory safety (e.g., null pointer dereferences)
+ * User-specified assertions (i.e., `assert!(...)`)
+ * The absence of panics (e.g., out-of-bounds accesses)
```

Comment:
Is an out-of-bounds access a panic or a memory safety issue? I think Kani detects it as a reachable panic. Maybe, to avoid the ambiguity we switch the example for absence of panics to `unwrap()` on `None`.